api/src/main/java/org/openmrs/module/registrationcore/api/mpi/common/MpiPatientImporter.java

```diff
@@ -0,0 +1,16 @@
+package org.openmrs.module.registrationcore.api.mpi.common;
+
+import org.openmrs.Patient;
+
+/**
+ * This service perform import patient from MPI server and convert it to local patient.
+ */
+public interface MpiPatientImporter {
+
+    /**
+     * Perform query to mpi server & convert it to local patient.
+     * @param patientId patient identifier
+     * @return converted patient
+     */
+    Patient importMpiPatient(String patientId);
```

Comment:
As I mentioned in an above comment, this should be called "fetchMpiPatient" since this method does not actually ingest the patient.